packages/roosterjs-editor-api/lib/format/setFontWeight.ts

```diff
@@ -0,0 +1,18 @@
+import applyListItemStyleWrap from '../utils/applyListItemWrap';
```

Comment:
suggest move this function to where it is used and do not export it for now. We can export it later when we really need.

It is weird that it can pass the build without marking it is @internal.